code/datums/spells/infinite_guns.dm

```diff
@@ -17,5 +19,41 @@
 		C.drop_item()
 		C.swap_hand()
 		C.drop_item()
-		var/obj/item/gun/projectile/shotgun/boltaction/enchanted/GUN = new
+		var/obj/item/gun/projectile/shotgun/boltaction/enchanted/GUN = new gun_type()
```

Comment:
```suggestion
		var/obj/item/gun/projectile/shotgun/boltaction/enchanted/GUN = new gun_type
```

Reply:
Parenthesis make it clearer imo.

Reply:
it's null either way, this is just a more obvious null